docs/specs/clients/sign/README.md

```diff
@@ -20,6 +20,28 @@ This swipe puts the dapp (usually in a browser window) on the background which m
 
 Thus if the dapp never receives the pairing response that means it also will never publish the session proposal meaning that the wallet is redirected with the URI but won’t prompt the user for approval.
 
-### Goal
+#### Goal
 
 The solution is to find a protocol change that inherits the same capability of v1.0 to propose session and pairing simultaneously but also persists the decoupled property of v2.0 protocol
+
+## Wallet Authentication Overview
+
+### Description
+
+Sign API provides a simple and lean interface to enable applications to authenticate wallet users to login/signin into applications with their wallets by automatically signing an authentication message.
+
+### Context
+
+Sign API's wallet authentication would be used for wallet users to login in a single step into existing websites or applications that would normally require an email and password login or a social login such as Facebook, Google, etc
+
+Sign API's wallet authentication is a one-click passwordless authentication provider for any website or application.
+
+### Goals
+
+A user can login into any website or application with their blockchain account.
+
+An app does not need to maintain a persisted connection with the wallet once authenticated.
```

Comment:
we could mention sessions here too

Reply:
I agree with Bartek, it'd be clearer if we mention sessions.

```suggestion
An app does not need to maintain a persisted connection with the wallet once authenticated thanks to a session guaranteeing ownership of the account.
```